game/theater/controlpoint.py

```diff
@@ -1143,10 +1184,15 @@ def mission_types(self, for_player: bool) -> Iterator[FlightType]:
 
     @property
     def total_aircraft_parking(self) -> int:
-        return 0
+        return len(self.helipads)
 
     def can_operate(self, aircraft: AircraftType) -> bool:
-        return False
+        print(aircraft)
+        print(aircraft.helicopter)
```

Comment:
Delete the prints. If there's useful logging to do, `logging.debug` with a useful message, but I think this probably isn't useful to keep?

Reply:
Yep, forgot those as well 😖

Reply:
Resolved in 9ea1edf9dbc8b21c7204ca6a71eab847ddc63522